tests/test_packages/test_skills_integration/test_ml_skills.py

```diff
@@ -41,12 +41,8 @@
 
 
 def _is_not_tensorflow_installed():
-    try:
-        import tensorflow  # noqa
-
-        return False
-    except ImportError:
-        return True
+    tf_spec = importlib.util.find_spec("tensorflow")
+    return tf_spec is None
```

Comment:
for some reason, importing `tensorflow` caused stability problems to these tests. In particular, tests were ignored even if they failed.

With this workaround, we only check the presence of the module without importing it.